charts/invenio/values.yaml

```diff
@@ -105,6 +105,21 @@ invenio:
     secretKeys:
       usernameKey: "DATACITE_USERNAME"
       passwordKey: "DATACITE_PASSWORD"
+    ## @param invenio.datacite.prefix DataCite DOI prefix
+    ##
+    prefix: ""
+    ## @param invenio.datacite.testMode DataCite test mode enabled.
+    ## This has to be a string value, invenio will evaluate this as a python expresion an transform it into a boolean.
+    ## Example:
+    ## testMode: "False"
+    ##
+    testMode: ""
+    ## @param invenio.datacite.format A string used for formatting the DOI.
+    ##
+    format: ""
+    ## @param invenio.datacite.centerSymbol DataCite data center symbol.
+    ##
+    centerSymbol: ""
```

Comment:
Sorry for my ignorance, but this reads like BIOS tool tips for me. 😁

> datacite.centerSymbol DataCite data center symbol.

Can we improve the comment with some info about what a "DataCite data center symbol" is?

Also, right now we call it both "data center symbol" and just "centerSymbol". Let's consolidate that.

Reply:
😂 
For all these variables, I just copied the first line of the config from Invenio-RDM-Records, in this case, https://github.com/inveniosoftware/invenio-rdm-records/blob/master/invenio_rdm_records/config.py#L481.

I looked into linking to docs but ... https://invenio-rdm-records.readthedocs.io/en/latest/configuration.html and didn't want to link to code directly because lines might change in the future.

Reply:
Ah, another case of the good ol' _undocumented parameters_ – nice!

My primary suggestion is that we copy or get inspired by the code you linked, and write our own comments based on that.

In second plance, I think a permalink to the code file is fine.

If you disagree with both of those, having no comments is not a deal breaker for me. At least that's easy to maintain. 😄

Reply:
I added a link to the general RDM documentation at the top (I didn't think about this one before) and also referenced the config variable that each key will set.
I think it's much better than before now 😉

Reply:
Big improvement. 👍